packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -659,13 +659,13 @@ class _InteractiveViewerState extends State<InteractiveViewer> with TickerProvid
 
   // Returns true iff the given _GestureType is enabled.
   bool _gestureIsSupported(_GestureType gestureType) {
-    if (_gestureType == _GestureType.pan && !widget.panEnabled) {
+    if (gestureType == _GestureType.pan && !widget.panEnabled) {
```

Comment:
nit: maybe use `switch` instead?

Reply:
That seems simpler to me, will do 👍

Reply:
I think using `default` clause is discouraged.